### ISSUE TITLE
Deployed a new full page web ui connected to my Lex chat bot but getting this message

### ISSUE THREAD

**Issue by realdanielbyrne** [aws-samples/aws-lex-web-ui#71]
Sorry, I was unable to process your message. Try again later.

Any idea as to how to debug this.  The stack creation proceeded normally.

Thank you,

**Comment by rstrahan** [comment]
Can you use Developer Tools in your browser to see if there are clues in the browser console log?

**Comment by RedwineDude** [comment]
Hi

I have tried this several times, with a few different Lex Bots. They're all built and published and work within Amazon Lex okay. I follow all the steps in this procedure successfully, all the parts are created okay and the chatbot fires up okay, but then when I try to use the bot in either my website or the WebAppUrl, I get the same messge regardless of what I type in:-

"Sorry, I was unable to process your message. Try again later."

or if I say "Hi" via the microphone:-

"Sorry, I had an error handling this conversation."

I am sure it is something very simple that I have overlooked.
